tcms_junit_plugin/__init__.py

```diff
@@ -93,10 +105,13 @@ def main(argv):
     )
     parser.add_argument('-v', '--verbose', dest='verbose', action='store_true',
                         help='Print information about created TP/TR records')
+    parser.add_argument('--summary-template', dest='summary_template', type=str,
+                        help='Template to convert testcase to summary, eg %(default)s.',
+                        default='${classname}.${name}')
```

Comment:
or maybe this value can become the default in `__init__` as well. Again for the sake of others inheriting from this class.

Reply:
Have gone for this suggestion of defaulting for both direct inheritors of the Plugin class and for people using the commandline arguments